test/controllers/api/jobs_controller_test.rb

```diff
@@ -72,6 +74,8 @@ def doorkeeper_token=(t)
     job.wont_be_nil
     job.tasks.size.must_equal 1
 
+    job.original_format.must_equal 'mp3'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.